pandas/core/tools/datetimes.py

```diff
@@ -385,6 +387,8 @@ def _convert_listlike_datetimes(
     """
     if isinstance(arg, (list, tuple)):
         arg = np.array(arg, dtype="O")
+    elif isinstance(arg, PandasArray):
+        arg = np.array(arg)
```

Comment:
Which test hits this?

I ran
```
pytest pandas/tests/tools/test_to_datetime.py \
pandas/tests/io/parser/test_parse_dates.py \
pandas/tests/io/parser/usecols/test_parse_dates.py \
pandas/tests/indexes/datetimes/test_constructors.py \
pandas/tests/indexes/datetimes/test_scalar_compat.py \
pandas/tests/series/test_constructors.py \
pandas/tests/tslibs
```
and couldn't hit it

Reply:
if i disable this line we fail `pandas/tests/arrays/test_datetimelike.py::test_from_pandas_array[M8[ns]] - TypeError: Cannot convert PandasDtype to numpy.dtype`